.github/workflows/workflow.yaml

```diff
@@ -0,0 +1,153 @@
+name: Workflow for rke-tools
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+  pull_request:
+    branches:
+      - master
+
+env:
+  IMAGE: rancher/rke-tools
+
+jobs:
+  ci:
+    permissions:
+      contents: read
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    container: 
+      image: rancher/dapper:v0.6.0
+    steps:
+      - name: Force Install GIT latest
+        run: |
+          apk add git --update-cache
+          git --version
+          which git
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Build
+        run: |
+          dapper ci
+      - name: Upload rke-etcd-backup artifact
+        if: github.event_name == 'push' && github.ref_type == 'tag'
```

Comment:
Again I think breaking these steps up into separate files would be easier to maintain, but this is more of a nit

Reply:
I have separated out fossa scans already. I dont think anything else in this workflow can be separated into a different file.